SerialPrograms/Source/PokemonSV/Programs/Eggs/PokemonSV_EggRoutines.h

```diff
@@ -103,6 +104,13 @@ bool check_baby_info(
     Pokemon::EggHatchAction& action
 );
 
+//This is a similar to check_baby_info, except for Stats Reset
+bool check_stats_reset_info(

```

Comment:
Since this is separate from eggs, let's not put it in the same file as the eggs.